src/biogeochem/CNDriverMod.F90

```diff
@@ -627,25 +638,29 @@ subroutine CNDriverNoLeaching(bounds,
     ! Update all prognostic carbon state variables (except for gap-phase mortality and fire fluxes)
     call CStateUpdate1( num_soilc, filter_soilc, num_soilp, filter_soilp, &
```

Comment:
Yes, this is an issue I see too. It kind of goes back to this broader issue with what we call CN right?  With fates active, we want it to coordinate with soil BGC, but be used an alternative to vegetation BGC. Yet, in the future, it could be possible that even with fates on, these CN veg routines could be doing things on non fates columns, such as crop, or if there was desire to have big-leaf and fates veg active simultaneously in different geographic regions.